pkg/game/elements/placed_tile.go

```diff
@@ -68,6 +129,15 @@ type LegalMove struct {
 	Meeple MeeplePlacement
 }
 
+// note: this is implemented to avoid inheriting the implementation
+// but is just a simple passthrough and its output may not be a legal move.
+func (move LegalMove) Rotate(rotations uint) LegalMove {
```

Comment:
I have mixed feelings about LegalMove.Rotate() not returning a legal move. Is this really unavoidable?

Reply:
It might be an merge notification. I mean it was already accepted in kuczys request. I haven't written this part XD

Reply:
Oh alright, sorry about that then.

Reply:
This one was wrongly marked as resolved, as it overrides actual implementation.
Current implementation looks like this https://github.com/YetAnotherSpieskowcy/Carcassonne-Engine/blob/1cee3142eb428c51b54b63509090ca6a16461b81/pkg/game/elements/placed_tile.go#L53

Reply:
This happened because you are currently re-adding around 50 commits made by @Jackenmen that should not be part of this pull request

Reply:
change restored